deployer/README.md

```diff
@@ -372,6 +373,71 @@ for the main cluster.
 
 It is recommended to have one curator for each Elasticsearch cluster.
 
+### About the Deployer generated secrets
+
+Part of the installation process that is done by the logging deployer is to generate
+certificates and keys and make them available to the logging components by means
+of secrets.
+
+The secrets that the components make use of are:
+
+    logging-curator
+    logging-curator-ops
+    logging-elasticsearch
+    logging-fluentd
+    logging-kibana
+    logging-kibana-proxy
+
+#### logging-curator, logging-kibana, logging-fluentd
+
+These three components all have a `ca`, `key` and `cert` entries.  These are what
+are used for mutual TLS communication with Elasticsearch.
+  1. `ca` contains the certificate that Elasticsearch uses.
```

Comment:
I think it would be clearer to say it's a certificate for validating the other server and client certificates. ES uses it to validate client certs, the other components use it to validate ES's server cert.